modules/oe_multilingual_url_suffix/config/schema/oe_multilingual_url_suffix.schema.yml

```diff
@@ -0,0 +1,10 @@
+oe_multilingual_url_suffix.settings:
+  type: config_object
+  label: 'Language detection method based on url suffix'
```

Comment:
Let's use `URL` all capital in labels and descriptions.

Reply:
Done